packages/web-react/src/components/Toast/__tests__/ToastBarLink.test.tsx

```diff
@@ -0,0 +1,36 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import ToastBarLink from '../ToastBarLink';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+
+describe('ToastBarLink', () => {
+  restPropsTest((props) => <ToastBarLink {...props} id="test" />, 'a');
+
+  stylePropsTest(ToastBarLink);
+
+  restPropsTest(ToastBarLink, 'a');
+
+  beforeEach(() => {
+    render(<ToastBarLink href="example-href">Example action</ToastBarLink>);
```

Comment:
can we use valid value? :)

Reply:
I will still like to see some more valid value. 
```suggestion
    render(<ToastBarLink href="#example-href">Example action</ToastBarLink>);
```
and the `expect` has to be updated too, then. 

But it is just my nitpicking, feel free to ignore me and resolve this conversation.